internal/sql/repository/chartConfig/ChartRepository.go

```diff
@@ -73,6 +118,151 @@ type ChartRepositoryImpl struct {
 	dbConnection *pg.DB
 }
 
+func (repositoryImpl ChartRepositoryImpl) FindBatchOperationExample(task string) (*BatchOperationExample, error) {
+	batchOperationExample := &BatchOperationExample{}
+	err := repositoryImpl.dbConnection.
+		Model(batchOperationExample).Where("task LIKE ?", task).
+		Select()
+	return batchOperationExample, err
+}
+
+func (repositoryImpl ChartRepositoryImpl) FindBulkAppNameIsGlobal(appNameIncludes []string, appNameExcludes []string) ([]*App, error) {
+	var apps []*App
```

Comment:
we can optimize this query with https://stackoverflow.com/questions/4928054/postgresql-wildcard-like-for-any-of-a-list-of-words/4928415